src/pydata_sphinx_theme/assets/styles/sections/_sidebar-primary.scss

```diff
@@ -140,61 +141,106 @@
   .list-caption {
     list-style: none;
     padding-left: 0px;
-  }
-  li {
-    position: relative;
-    // If it has children, add a bit more padding to wrap the content to avoid
-    // overlapping with the <label>
-    &.has-children {
-      > .reference {
-        padding-right: 30px;
+
+    .nav {
+      display: block;
+    }
```

Comment:
Going by the comments in `toc-inpage.scss` that `display: none` seems to be related to https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/navigation.html#collapse-entire-toctree-captions-parts 

so that the user can make those collapsible or specify the level through the `config.py`

Reply:
I dig some digging on this. I decided that it probably makes most sense to remove the `.list-caption` selector from the in-page-TOC stylesheet. See commit 85f2ad1bf8a2a094c8563a742da0ecfa26c1fc2f.

Semantically, I don't think that the `.list-caption` class (introduced in #594) should have ever been added to that stylesheet. As far as I can tell, the class is only applied in the **left** sidebar, whereas the in-page-TOC stylesheet is for the right sidebar. So why was it added to that stylesheet? I'm not sure. Maybe it was to pick up some of the other rules in that block of styles? In any case, it doesn't seem necessary now, and removing `.list-caption` from that stylesheet allows me to remove the `display: block` override, as well as make that stylesheet more semantically coherent.

As for your hunch, @trallard, I think it's close. It looks to me that the CSS block in the in-page-TOC stylesheet is related to `show_toc_level`, which is similar to but different from `show_nav_level`. I think the first one applies to the right sidebar whereas the second applies to the left. See https://github.com/pydata/pydata-sphinx-theme/pull/256.

Reply:
Here's a screenshot of the website with:

- `.list-caption` removed from `_toc-inpage.scss`
- `show_nav_level` = 0
- `show_toc_level` = 4

![](https://github.com/pydata/pydata-sphinx-theme/assets/317883/c903b876-908d-4ff0-bf2d-02de172739cc)

It looks right to me. I also clicked around, and things looked right. As far as I can tell, the `.list-caption` is only used in the left sidebar when `show_nav_level` is set to 0.